eng/common/pipelines/templates/jobs/archetype-spec-gen-sdk.yml

```diff
@@ -86,12 +86,14 @@ jobs:
           Exit 1
         }
 
-        $sdkRepoNameForSpecGen = $sdkRepoName
-        if ($sdkRepoName.EndsWith('-for-net')) {
-          $sdkRepoNameForSpecGen = $sdkRepoName + "-track2"
+        if ($env:BUILD_REASON -eq 'PullRequest') {
+           $specRepoCommitVar = $env:BUILD_SOURCEBRANCH
```

Comment:
What does this variable end in "Var"?

Reply:
I didn't want to have same name as `parameter.SpecRepoCommit` though it's allowed.